src/main/api/2/studio-api-2.yaml

```diff
@@ -3860,6 +3860,86 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content/children_by_paths:
+    post:
+      tags:
+        - content
+      summary: Get list of children for given item paths
+      description: 'Required permission "get_children" and "site member"'
+      operationId: getChildrenByPaths
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                siteId:
+                  description: Site ID
+                  type: string
+                localeCode:
```

Comment:
Let's move the locale code inside the request array to allow for it to be different if required.